components/SegmentHeader.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <header>
+    <img 
+      v-if="imagePath"
+      :class="{ round: isImageRound }"
+      :src="imagePath"/>
+    <div 
+      v-if="name"
+      class="nameInformation">
+      <h1>
+        <span v-text="name.first"/><br/>
```

Comment:
Why use two spans instead of combinding them to a single string and adding v-text directly to h1